doc/source/user_guide/data_sources_tutorial.rst

```diff
@@ -0,0 +1,310 @@
+.. _user_guide_data_sources:
+
+===============
+DPF DataSources
+===============
+
+The DataSources operator is a class available as a submodule of the 'ansys.dpf.core'
+package. It contains files with the analysis results. So this object is used in
+the first basic step: define simulation data, by defining where it is located.
+
+This example will describe the functionalities and functions of the class DataSources
+and can be generalised for all the files formats accepted by Py-DPF:
+
+.. _list_extensions_solvers:
+**Supported apps by DPF and their related formats**
+
+
+   - MAPDL files: .rst, .mode, .rfrq, .rdsp
+   - LS-DYNA files: .d3plot, .binout
+   - Fluent files: CFF restart files(.cas/dat.h5) and Project files (.flprj)
+   - CFX files: CFF files(.cas/dat.cff.res) and Project files (.flprj)
+
```

Comment:
You will have to also add a list of other file types we support, which are not Ansys solver output files, such as `cgns`, `hdf5`, and a few others. We can discuss that as it is not that simple to describe, but it is very important